packages/node_modules/@node-red/editor-client/src/js/ui/view.js

```diff
@@ -2680,11 +2711,16 @@ RED.view = (function() {
                     var l = RED.utils.getNodeLabel(d);
                     if (d.resize || d.w === undefined) {
                         if (hideLabel) {
-                            d.w = node_height;
+                            d.w = 30;
                         } else {
                             d.w = Math.max(node_width,20*(Math.ceil((calculateTextWidth(l, "red-ui-flow-node-label", 50)+(d._def.inputs>0?7:0))/20)) );
                         }
                     }
+                    if (hideLabel) {
+                        node_height = 30;
+                    } else {
+                        node_height = 6 + 24 * convertLineBreakCharacter(l).length;
```

Comment:
`node_height` is a constant that **must not be changed**. `d.h` is the individual node's height that can be changed.

Please restore the use of `node_height` as a constant in the few places you have removed it.